main.go

```diff
@@ -44,6 +48,21 @@ import (
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
+
+	supportedSchemeReconciler = map[string]func(manager.Manager) error{
+		strings.ToLower(tensorflowv1.Kind): func(mgr manager.Manager) error {
```

Comment:
Maybe we do not need this ToLower?

`	flag.StringVar(&enabledSchemeStr, "enable-scheme", "all", "Enable scheme(s), separated by comma. "+
		"Now supporting TFJob, PyTorchJob, MXNetJob, XGBoostJob.")`

Reply:
Well, I suppose some user may use input like: `./manager --enable-scheme=tfjob,mxnetjob,pytorchjob`